### DISCUSSION TITLE
How to use scala-yaml for scala2？

### DISCUSSION THREAD

**Discussion by colin4124** [VirtusLab/scala-yaml#189] [General]
Document's example is all about scala 3, how about scala2 ?

**Comment by kpodsiad** [comment]
Hey @colin4124! There is no codec/encoder derivation for Scala 2 so you can only convert Strings to YAML AST. However, there is an integration with circe created by @armanbilge - https://github.com/armanbilge/circe-scala-yaml. Below I pasted [scala-cli](https://github.com/armanbilge/circe-scala-yaml) snippet with some usage of them both. 

```scala
//> using scala "2.13.10"
//> using lib "org.virtuslab::scala-yaml::0.0.6"
//> using lib "com.armanbilge::circe-scala-yaml::0.0.4"

import org.virtuslab.yaml._
import io.circe.scalayaml._

object Yaml extends App {
  val myYaml = """foo: bar"""

  println(parser.parse(myYaml))
  println()
  println(myYaml.asNode)
}
```

**Reply by colin4124** [reply]
Thanks so much, I use circe-scala-yaml directly, it works.
